### DISCUSSION TITLE
Delete old Events from postgres db

### DISCUSSION THREAD

**Discussion by wheeleran** [umami-software/umami#2052] [General]
Hi all. in Umami v1 I used to be able to delete old test Event data by navigating to the postgres db, then the psql shell, and run the following: ``` delete from event where event_name not in ('your-event-name', 'another-event-name', 'everything-to-not-delete');```
this doesn't seem to be working any more in v2? i've been fooling around  getting used to the new data attributes and i've got some crap to delete and can't. would appreciate any help!

**Comment by briancao** [comment]
The `event` table was renamed to `website_event`, so you'll have to modify your script accordingly.

`delete from website_event where event_name not in ('your-event-name', 'another-event-name', 'everything-to-not-delete');`

**Reply by wheeleran** [reply]
Thanks, @briancao. Could 'working with your data' not be a small section in the docs?

**Reply by briancao** [reply]
I just don't think we can catch all the unique use-cases, in a meaningful enough way, to document how users want to work and interact with their data outside of the umami application.

**Reply by wheeleran** [reply]
Understood, thanks.
